mscore/toolbuttonmenu.cpp

```diff
@@ -13,24 +13,25 @@
 
 namespace Ms {
 
-ToolButtonMenu::ToolButtonMenu(QString str,
-                     TYPES type,
+ToolButtonMenu::ToolButtonMenu(QString name,
                      QAction* defaultAction,
                      QActionGroup* alternativeActions,
-                     QWidget* parent) : AccessibleToolButton(parent, defaultAction)
+                     QWidget* parent,
+                     bool swapAction)
+      : AccessibleToolButton(parent, defaultAction)
       {
-      // if, and only if, type is ACTION_SWAPPED then the default action is also an alternative action.
-      Q_ASSERT((type == TYPES::ACTION_SWAPPED) == alternativeActions->actions().contains(defaultAction));
+      // does the default action count as one of the alternative actions?
+      Q_ASSERT(swapAction == alternativeActions->actions().contains(defaultAction));
 
-      _type = type;
+      _swapAction = swapAction;
       _alternativeActions = alternativeActions;
 
-      setMenu(new QMenu(str, this));
+      setMenu(new QMenu(name, this));
       menu()->setToolTipsVisible(true);
 
       setPopupMode(QToolButton::MenuButtonPopup);
```

Comment:
Changing this line to:
```c++
setPopupMode(QToolButton::InstantPopup);
```
would give the same effect as the accessibility commit, but it would also change the behaviour for sighted users.

Let me know which you prefer.